tests/bcr/MODULE.bazel.lock

```diff
@@ -1255,17 +1255,8 @@
       }
     },
     "@gazelle~override//:extensions.bzl%go_deps": {
-      "bzlTransitiveDigest": "ti6/wrYI3K/ecfLzekOUxKJVqza2UYx79wFadZG0AMQ=",
-      "accumulatedFileDigests": {
-        "@@//:go.mod": "8e62c686b94b37593b38766d425ceccbf86a9b07c0121feaaf42293050a42ae3",
-        "@@circl~1.3.3//:go.mod": "68b12e4662bb0639728490153ffc52d8bdd63be558bdd41bcb0d8f1eeeb03e41",
-        "@@//:go.sum": "05fce820627579dc40f60ac8fad81521932d2727e9ff6ab0247a661ccb124ecf",
-        "@@rules_go~0.41.0//:go.mod": "a7143f329c2a3e0b983ce74a96c0c25b0d0c59d236d75f7e1b069aadd988d55e",
-        "@@circl~1.3.3//:go.sum": "c895d918735c8cff9d877af70b1177c11b456e87b5a6a9aec025ddae0090300b",
-        "@@gazelle~override//:go.mod": "b35d94761afb0ec7afccf76735f437bd2396025fd02c6eb14d53cb81e7b0cba0",
-        "@@gazelle~override//:go.sum": "b75476f84cf95bfcbbebc71263437b9bc3c295c10fb4d316ae77cbd76a2c70cd",
-        "@@rules_go~0.41.0//:go.sum": "022d36c9ebcc7b5dee1e9b85b3da9c9f3a529ee6f979946d66e4955b8d54614a"
-      },
+      "bzlTransitiveDigest": "IdiouZbvhKAbTCt17T8F3TCf9s/An6guFF2G0wwz5BE=",
+      "accumulatedFileDigests": {},
```

Comment:
@fmeum Can you double check that this is safe? A bit unsure what's happening here since `tests/bcr/MODULE.bazel` is not changed.

Reply:
Actually, it appears to have fixed itself after I reset the contents of the file and ran `bazel mod tidy` again. Something interesting happened I think when I ran a `bazel test` locally from an older version of bazel.